frontend/src/Editor/QueryManager/Components/Preview.jsx

```diff
@@ -116,6 +128,43 @@ const Preview = ({ darkMode }) => {
                     {renderRawData()}
                   </div>
                 </Tab.Pane>
+                {queryPreviewData !== '' && kind === 'tooljetdb' && (
+                  <Tab.Pane eventKey="table" transition={false}>
+                    <div
+                      className={` raw-container preview-data-container overflow-x-scroll`}
+                      data-cy="preview-raw-data-container"
+                    >
+                      <Table stripped hover variant={darkMode && 'dark'} size="lg" className="py-2">
+                        <thead>
+                          <tr>
+                            {columns?.map((columnName, i) => (
+                              <th key={i} className=" p-2 text-nowrap ">
+                                {columnName}
+                              </th>
+                            ))}
+                          </tr>
+                        </thead>
+                        <tbody>
+                          {console.log('cdfvdcsxadfvgbrdfvcsxcfvgrdfcsxcfvdcxsdfvc', queryPreviewData)}
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/ToolJet/ToolJet/pull/10065#issuecomment-2163104325>#17d927</a><span> - 06/12/2024, 02:07 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> There is an active console.log statement within the render logic. This can lead to performance issues and unnecessary logging in production.
</div>
<div id="fix">
<b>Fix:</b> Remove the console.log statement to avoid performance issues and unnecessary logging in production.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div>
<pre><code>
 @@ -148 +148 @@
 -                          {console.log('cdfvdcsxadfvgbrdfvcsxcfvgrdfcsxcfvdcxsdfvc', queryPreviewData)}
</code></pre>
</div>
</details>
</div>
<br>


</div>